tests/functional/spec/question_description.spec.js

```diff
@@ -0,0 +1,8 @@
+import NameBlockPage from "../generated_pages/question_description/name-block.page.js";
+
+describe("Question description", () => {
+  it("Given a question description has been set in the schema as an array, it is displayed correctly as a newline-separated string", () => {
+    browser.openQuestionnaire("test_question_description.json");
+    expect($(NameBlockPage.questionTitle()).getText()).to.contain("Answer the question\nGo on");
```

Comment:
I didn’t get as far as inspecting the `<p>` elements for the description test if necessary I'll work on it but at least this gets hold of the content